mail_base/README.rst

```diff
@@ -0,0 +1,11 @@
+Mail Base
+=========
+
+Module contain common code for other mail modules
+
+Usage
+-----
+
```

Comment:
To use this module you need either install module that depends on it or create new module.